WordPressAuthenticator/Unified Auth/View Related/Sign up/UnifiedSignUpViewController.swift

```diff
@@ -44,7 +46,133 @@ class UnifiedSignUpViewController: LoginViewController {
         view.backgroundColor = unifiedBackgroundColor
     }
 
+    /// Style individual ViewController status bars.
+    ///
     override var preferredStatusBarStyle: UIStatusBarStyle {
         return WordPressAuthenticator.shared.unifiedStyle?.statusBarStyle ?? WordPressAuthenticator.shared.style.statusBarStyle
     }
+
+    /// Override the title on 'submit' button
+    ///
+    override func localizePrimaryButton() {
+        submitButton?.setTitle(WordPressAuthenticator.shared.displayStrings.magicLinkButtonTitle, for: .normal)
+        submitButton?.setTitle(WordPressAuthenticator.shared.displayStrings.magicLinkButtonTitle, for: .highlighted)
```

Comment:
since the title is the same, you don't need to specify it for the `.highlighted` state: https://developer.apple.com/documentation/uikit/uibutton/1624018-settitle